tensorboard/backend/application.py

```diff
@@ -155,6 +156,8 @@ def __init__(self, logdir, plugins, multiplexer, reload_interval):
       try:
         plugin_apps = plugin.get_plugin_apps()
       except Exception as e:  # pylint: disable=broad-except
+        if plugin.plugin_name == 'core':
```

Comment:
Yes. It's pretty fatal if it doesn't initialize. To be honest, I'd like to remove this error suppression for all plugins.